framework/quota/src/main/java/org/apache/cloudstack/quota/dao/QuotaAccountDaoImpl.java

```diff
@@ -36,7 +38,15 @@ public class QuotaAccountDaoImpl extends GenericDaoBase<QuotaAccountVO, Long> im
 
     @Override
     public List<QuotaAccountVO> listAllQuotaAccount() {
-        return listAllQuotaAccount(null, null).first();
+        List<QuotaAccountVO> accountsWithQuotaEnabled = new ArrayList<>();
+        for (QuotaAccountVO account : listAllQuotaAccount(null, null).first()) {
+            if (Boolean.TRUE.equals(getQuotaAccountEnabled(account.getAccountId()))) {
```

Comment:
We can use `org.apache.commons.lang3.BooleanUtils#isTrue()` here: 

```suggestion
            if (BooleanUtils.isTrue(getQuotaAccountEnabled(account.getAccountId()))) {
```

Reply:
@GutoVeronezi how is this different (/better) ?

Reply:
@DaanHoogland nothing in special in this case, just demonstrating a library that facades the operation (better to read IMHO).